robot_lib.py

```diff
@@ -26,21 +26,22 @@ def mountRobotSample(gov_robot, puck_pos, pin_pos, samp_id, **kwargs):
   if status != MOUNT_STEP_SUCCESSFUL:
       return status
   status = robot.postMount(gov_robot, puck_pos, pin_pos, samp_id)
-  return MOUNT_SUCCESSFUL  # TODO hard-coded for testing
+  return status
 
 def unmountRobotSample(gov_robot, puck_pos, pin_pos, samp_id):
   status = robot.preUnmount(gov_robot, puck_pos, pin_pos, samp_id)
   if status != UNMOUNT_STEP_SUCCESSFUL:
       return status
+  
+  status = UNMOUNT_SUCCESSFUL
```

Comment:
are these necessary? status is defined for both cases below, it shouldn't be necessary to be defined before then

Reply:
You're right fixed in https://github.com/NSLS-II/lsdc/pull/318/commits/64425a2ff92288aa04b06134298f0e168acaf99d